src/components/map/courseInfo/BottomCourseInfoModal/CreateGhosty.tsx

```
@@ -0,0 +1,222 @@
+        setStep((prev) => Math.min((prev ?? 0) + 1, steps.length - 1));
+
+    const steps = [
+        <StepCheckExperience state={state} dispatch={dispatch} />,
+        <StepSelectGhosty state={state} dispatch={dispatch} />,
+        <StepConditionCheck state={state} dispatch={dispatch} />,
+        <StepCreating />,
+    ];
+
+    useEffect(() => {
+        (async () => {
+            const hasRunHistory = false;
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)